Fabric.Identity.API/scripts/Install-Identity-Discovery-IdPSS.ps1

```diff
@@ -0,0 +1,62 @@
+﻿#Requires -RunAsAdministrator
+#Requires -Version 5.1
+#Requires -Modules PowerShellGet, PackageManagement
+
+param(
+    [PSCredential] $credential,
```

Comment:
Should there be multiple options for credentials now? One for Discovery and for Identity/IdPSS?

Reply:
i see what you mean, ill fix that